ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -376,6 +394,51 @@ class GraphsContainer extends React.Component {
                   />
                 ))}
 
+              {changelogData.length > 0 &&
+               <VictoryBar
```

Comment:
The ordering of this matters, so that might be why this is hard to click on - it might need to sit on top of other elements.

Reply:
I think since these are lines it might make more sense to use `VictoryLine` instead.